Apps/Encounter/src/Models/EncounterModel.cs

```diff
@@ -63,7 +64,7 @@ public static EncounterModel FromODRClaimModel(Claim model)
         {
             return new EncounterModel()
             {
-                Id = model.ClaimId.ToString(),
+                Id = HashGenerator.ComputeHashToGuid($"{model.ServiceDate:yyyyMMdd}{model.SpecialtyDesc}{model.PractitionerName}{model.LocationName}").ToString(),
```

Comment:
I think we also need to qualify it by the clinic full address

Reply:
@ninosamson can you clarify this?

Reply:
per @ninosamson 's confirmation, I will include the full address.